web-ng/src/app/services/drawing-tools/drawing-tools.service.ts

```diff
@@ -20,18 +20,18 @@ import { BehaviorSubject } from 'rxjs';
 @Injectable({
   providedIn: 'root',
 })
-export class DrawingKitService {
+export class DrawingToolsService {
   private editMode$: BehaviorSubject<EditMode> = new BehaviorSubject<EditMode>(
     EditMode.None
   );
-  private layerId = '';
+  private selectedLayerId: string | undefined;
```

Comment:
Does `string?` work here and below instead of `| undefined`?

Reply:
No the syntax doesn't work